base-notebook/start-notebook.sh

```diff
@@ -12,8 +12,12 @@ fi
 if [[ ! -z "${JUPYTERHUB_API_TOKEN}" ]]; then
     # launched by JupyterHub, use single-user entrypoint
     exec /usr/local/bin/start-singleuser.sh "$@"
-elif [[ ! -z "${JUPYTER_ENABLE_LAB}" ]]; then
-    . /usr/local/bin/start.sh $wrapper jupyter lab "$@"
-else
```

Comment:
Perhaps we should keep the default like it is for a period of time, but emit a deprecation warning that links to https://github.com/jupyter/notebook#notice and suggests the new default will become jupyterlab by some soft date X months out?

Maybe we should mirror that message into the root README of this repo too?

When we hit X months, we can then change the default with some assurance that we made a reasonable attempt at notifying users of the breaking change, above and beyond suggesting "don't use latest if you want stability".

Reply:
👍 Thanks @parente very good suggestion. So I will change the default (no variable) back to Jupyter Notebook and change the deprecation warning to add the link. I will also put this link into the main README and into the doc.
In addition to these suggestions, should we keep and document the variable `JUPYTER_ENABLE_NB`? It will be a way to prepare the next step?

Reply:
Sure though it might be a bit confusing to new users in the interim. What happens if enable lab and enable nb are both set?

Reply:
In this case I will make JupyterLab win. I will add this test case in the test already written to be clear. The advantage of having it available as soon as possible is to permit to users to use it if they do not want to switch to JupyterLab during the next step.

Reply:
You're right I will simply drop the `JUPYTER_ENABLE_NB` variable for now, because it's useless in this first phase. So I will update this PR to add the warning messages and adapt the test I have written.